docs/diracdoctools/cmd/commandReference.py

```diff
@@ -116,16 +119,15 @@ def createFiles(self, sectionDict):
     with open(os.path.join(sectionPath, 'index.rst')) as indexFile:
       commandList = indexFile.read().replace('\n', '')
 
+    futures = []
+    with ThreadPoolExecutor() as pool:
+      for script in sectionDict[SCRIPTS] + sectionDict[MANUAL]:
+        futures.append(pool.submit(self.createScriptDocFiles, script, sectionPath, sectionDict))
+
     missingCommands = []
-    for script in sectionDict[SCRIPTS] + sectionDict[MANUAL]:
-      scriptName = os.path.basename(script)
-      if scriptName.endswith('.py'):
-        scriptName = scriptName[:-3]
-      scriptName = scriptName.replace("_", "-")
-      prefix = sectionDict[PREFIX].lower()
-      prefix = prefix + '_' if prefix else ''
-      if self.createScriptDocFiles(script, sectionPath, scriptName, referencePrefix=prefix) and \
-         scriptName not in commandList:
+    for future in futures:
+      scriptName, createdScriptDocs = future.result()
```

Comment:
This is a left over after the tidying, isn't it? `future.result()` isn't a tuple, but just the `True/False` from `createScriptDocFiles`, right?

Reply:
Nice spot! It still needs to be a tuple for the `scriptName not in commandList`.

It should be fixed now, it's being built in https://readthedocs.org/projects/cburrdirac/builds/13199424/